oaknational/Oak-Web-Application
# PUPIL-485: Prompt pupils to fill in questions when they attempt to proceed

Music year: [1978](https://www.youtube.com/watch?v=M-NW3qLcUlA)


## TODO
- [x] Bump oak-components with https://github.com/oaknational/oak-components/pull/132
- [x] Bump oak-components with https://github.com/oaknational/oak-components/pull/136

## Description

- When a pupil submits an answer to a question without attempting to answer it:
  - Displays a tooltip tailored to the question type instructing them to do so
  - Displays the question UI in a "highlighted" state
- "Check" button is no longer disabled.

## How to test

1. Go to https://deploy-preview-2321--oak-web-application.netlify.thenational.academy/pupils/lessons/joining-using-and
2. Click on "Starter quiz"
3. Click "Check" on the questions without first answering them
4. You should see a tooltip and highlighted UI

## Screenshots

https://github.com/oaknational/Oak-Web-Application/assets/122096/a2099968-eeea-485c-accf-b59879e0d766

## Checklist
